src/Realm.C

```diff
@@ -1928,7 +1942,46 @@ Realm::create_output_mesh()
       if (fileid++ > 0) oname += "-s" + fileid_ss.str();
     }
 
-    resultsFileIndex_ = ioBroker_->create_output_mesh( oname, stk::io::WRITE_RESULTS, *outputInfo_->outputPropertyManager_);
+
+    if(!outputInfo_->catalystFileName_.empty()||
```

Comment:
Is the concept here that one obtains either Catalyst **or** standard output? Is there a manner to support both in a single simulation? I think that would be most preferable.